cg/services/order_validation_service/validators/inter_field/rules.py

```diff
@@ -73,6 +75,20 @@ def validate_buffers_are_allowed(order: TomteOrder) -> list[CaseSampleError]:
     return errors
 
 
+def validate_concentration_required_if_skip_rc(
+    order: TomteOrder,
+) -> list[ConcentrationRequiredIfSkipRCError]:
+    errors = []
+    for case in order.cases:
+        for sample in case.samples:
+            if is_concentration_missing(sample, order.skip_reception_control):
```

Comment:
I'd move the `order.skip_reception_control` further up and just return early if it is not set for slightly more readable code. Something like
```python
def validate_concentration_required_if_skip_rc(
    order: TomteOrder,
) -> list[ConcentrationRequiredIfSkipRCError]:
    errors = []
    for case in order.cases:
        for sample in case.samples:
            if is_concentration_missing(sample, order.skip_reception_control):
```

Reply:
I agree that this makes more sense 👍

Reply:
Moved it